_posts/2020-04-29-cockpit-218.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 218
+author: pitti
+date: 2020-04-29
+tags: cockpit linux
+slug: cockpit-218
+category: release
+summary: Cockpit with Services page improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 218.
+
+### Services: Better accessibility and mobile support
+
+The Services page got rewritten using [PatternFly 4](https://www.patternfly.org/v4/) for improved accessibility. It now also works properly in mobile mode:
+
+![Services page mobile mode](/images/services-pf4-mobile.png)
+
+### Overview: Display uptime
+
+The "System Information" card on the Overview page shows the machine's uptime, i. e. how long it has been running since the last boot.
+
+![Overview uptime](/images/overview-uptime.png)
+
+Thanks to Martin Schiffner for this contribution!
+
+### Disable idle timeout by default
+
+As a response to user feedback, the automatic session timeout after 15 minutes that got introduced into [Cockpit 219](https://cockpit-project.org/blog/cockpit-209.html) is now disabled by default. As Cockpit is never the primary session on a device, it is sufficient to let the window manager on Desktops or the screen lock on mobile devices take care of protecting the session.
+
+The idle timeout can still be enabled in [cockpit.conf](https://cockpit-project.org/guide/latest/cockpit.conf.5.html), and any existing explicit setting continues to work as before.
+
+### Support building without polkit
+
+Cockpit integrates a [polkit](https://www.freedesktop.org/software/polkit/docs/latest/polkit.8.html) agent, so that operations to D-Bus services like firewalld or udisks can be allowed to users who are not full administrators. However, polkit has some large dependencies like mozjs which are undesirable for embedded systems. With this Cockpit version it is now possible to build without the polkit libraries; this will disable the polkit agent.
```

Comment:
```suggestion
Cockpit integrates a [polkit](https://www.freedesktop.org/software/polkit/docs/latest/polkit.8.html) agent, to handle D-Bus services like firewalld and udisks for users who are not full administrators. However, polkit has some large dependencies like mozjs which may be undesirable for embedded systems. With this Cockpit version, building without the polkit libraries is possible. This will, of course, disable the polkit agent within Cockpit sessions.
```

Reply:
I like Garrett's version, but I'd add "authentication for" after "to handle".

I'd also remove the "However" here, because the following text doesn't stand in contrast to the sentence preceding.  Perhaps "Unfortunately"?